README.md

```diff
@@ -99,6 +98,45 @@ tuner.run()
 
 The above example runs ASHA with 4 asynchronous workers on a local machine.
 
+## Supported HPO methods
+
+The following hyperparameter optimization (HPO) methods are available in Syne Tune:
+
+Method | Reference | Searcher | Asynchronous? | Multi-fidelity? | Transfer? 
+:--- | :---: | :---: | :---: | :---: | :---: 
+Grid Search | Someone (sometime) | deterministic | no | no | no 
+Random Search | Bergstra, et al. (2011) | random | yes | no | no 
+Bayesian Optimization | Snoek, et al. (2012) | model-based | yes | no | no 
+BORE | Tiao, et al. (2021) | model-based | yes | no | no 
+MedianStoppingRule | Golovin, et al. (2017) | any | yes | yes | no 
+SyncHyperband | Li, et al. (2018) | random | no | yes | no 
+SyncBOHB | Falkner, et al. (2018) | model-based | no | yes | no 
+SyncMOBSTER | Klein, et al. (2020) | model-based | no | yes | no 
+ASHA | Li, et al. (2019) | random | yes | yes | no 
+BOHB | Falkner, et al. (2018) | model-based | yes | yes | no 
+MOBSTER | Klein, et al. (2020) | model-based | yes | yes | no 
+DEHB | Awad, et al. (2021) | random | no | yes | no 
+HyperTune | Li, et al. (2022) | model-based | yes | yes | no 
+PASHA | Bohdal, et al. (2022)| random | yes | yes | no 
+REA | Real, et al. (2019) | model-based | yes | no | no 
+PBT | Jaderberg, et al. (2017) | model-based | no | yes | no 
+ZeroShotTransfer | Wistuba, et al. (2015) | deterministic | yes | no | yes 
+AHSACTS | Salinas, et al. (2021)| random | yes | yes | yes 
+RUSH | Zappella, et al. (2021)| random | yes | yes | yes 
+
+The searchers fall into three broad categories, **deterministic**, **random**, and **model-based**. The random searchers sample candidate hyperparameter configurations uniformly at random, while the model-based searchers sample them non-uniformly at random, according to a model (e.g., Gaussian process, density ration estimator, etc.) and an acquisition function, or by applying an evolutionary algorithm.
+
+Syne Tune also supports [BoTorch](https://github.com/pytorch/botorch) searchers.
```

Comment:
Can we add this one to the table rather? (to avoid having methods either in the table or in the text)

We can put Botorch with https://proceedings.neurips.cc/paper/2020/file/f5b1b89d98b7286673128a5fb112cb9a-Paper.pdf as the reference.

Reply:
Somebody needs to update this, because I am getting warnings now that we use deprecated API.

Reply:
I deliberately left BoTorch out of the table because it is an ensemble of methods and not just one.